<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchDialog</class>
 <widget class="QDialog" name="BatchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Batch Conversion</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbSourceFiles">
     <property name="text">
      <string>Source Files:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="lwSourceFiles">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl1">
     <item>
      <widget class="QPushButton" name="pbRemove">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbRemoveAll">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddFiles">
       <property name="text">
        <string>Add Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbAddDirectory">
       <property name="text">
        <string>Add Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbIncludeSubdirs">
       <property name="text">
        <string>Include Subdirs</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lbLog">
     <property name="text">
      <string>Log:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="pteLog">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hl2">
     <item>
      <layout class="QVBoxLayout" name="vl1">
       <item>
        <layout class="QHBoxLayout" name="hl4">
         <item>
          <widget class="QLabel" name="lbOutputFileFormat">
           <property name="text">
            <string>Output File Format:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cobOutputFileFormat">
           <property name="maxVisibleItems">
            <number>20</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hs1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl3">
         <item>
          <widget class="QLabel" name="lbOutputDirectory">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Output Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="hl6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="leOutputDirectory">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbChooseOutputFolder">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="maximumSize">
              <size>
               <width>25</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="hl5">
         <item>
          <widget class="QCheckBox" name="cbOutputToSourceDirs">
           <property name="text">
            <string>Output To Source Directories</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cbDeleteOriginals">
           <property name="text">
            <string>Delete Original Files</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hs2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="vl2">
       <item>
        <widget class="QPushButton" name="pbCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbStart">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
